va/va.h

```diff
@@ -530,7 +530,14 @@ typedef enum {
     VAProfileAV1Profile1                = 33,
     VAProfileHEVCSccMain444_10          = 34,
     /** \brief Profile ID used for protected video playback. */
-    VAProfileProtected                  = 35
+    VAProfileProtected                  = 35,
+
+    /**
+     * @note 'VA_PROFILE_H264_HIGH_10' is an conditional compiling option
+     * for 'VAProfileH264High10' support.
+     */
+#define VA_PROFILE_H264_HIGH_10
```

Comment:
I dont think that libva need add such macro,  and it is always defined.  please remove it. if driver want it, it should be defined inside driver.

Reply:
It was original wish to let mediadriver and applications such as ffmpeg support this new feature while still make backward compatibility with old version of libva.

Never mind，I can wait the new release and then merge corresponding code to FFmpeg by using 'VA_CHECK_VERSION'